scene/gui/color_picker.cpp

```diff
@@ -247,15 +247,19 @@ void ColorPicker::_update_color(bool p_update_sliders) {
 }
 
 void ColorPicker::_update_presets() {
+	presetsPerRow = 10;
 	Size2 size = bt_add_preset->get_size();
-	Size2 preset_size = Size2(size.width * presets.size(), size.height);
+	Size2 preset_size = Size2(MIN(size.width * presets.size(), presetsPerRow * size.width), size.height * (Math::ceil((float)presets.size() / presetsPerRow)));
 	preset->set_custom_minimum_size(preset_size);
-
-	preset->draw_texture_rect(get_icon("preset_bg", "ColorPicker"), Rect2(Point2(), preset_size), true);
+	preset_container->set_custom_minimum_size(preset_size);
+	preset->draw_rect(Rect2(Point2(), preset_size), Color::named("transparent"));
```

Comment:
In the backend, I'd say it's better to use `Color(1, 1, 1, 0)` than to allocate a string each time for the named constructor. Named colors are mainly provided as a convenience for users of the scripting language, not so much for the backend code.